agent/api/task/task_linux.go

```diff
@@ -148,18 +148,13 @@ func (task *Task) buildImplicitLinuxCPUSpec() specs.LinuxCPU {
 	// aggregate container CPU shares when present
 	var taskCPUShares uint64
 	for _, container := range task.Containers {
-		if container.CPU > 0 {
+		if container.CPU < minimumCPUShare {
```

Comment:
if I have 2 containers with `container.CPU = 0`. then `taskCPUShares` would become 4?

Reply:
Yes - I believe that should be the correct behavior because we later on [bump](https://github.com/aws/amazon-ecs-agent/blob/6f6e351b3b266984e257a88042f73ef4b9469d6b/agent/api/task/task_unsupported.go#L53) container CPU shares to 2, if the value specified in task def is smaller (or unspecified). So if you have 2 containers they will both have CPUShares=2 when passed to docker, which makes sense for the task CPU share to be 4 in that case.